lib/generic/read_png.cpp

```diff
@@ -0,0 +1,106 @@
+/// \file
+/// Maintainer: Felice Serena
+///
+///
+
+#include "read_png.h"
+
+#include <png.h>
+#include <boost/log/trivial.hpp>
+
+namespace MouseTrack {
+
+Picture read_png_normalized(const std::string& path){
+    auto mat = read_png(path);
+    Picture result = mat.cast<double>();
+    result /= 256.0;
```

Comment:
Isn't the max pixel value 255, so we should divide by 255?

Reply:
you're right, I'll adjust it.